src/components/Tabs.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => (
+  <ul>
+    {tabs.map(({ id, title, content }) => (
+      <li key={id}>
+        <button
+          type="button"
+          href="#"
+          onClick={() => onTabSelected(id)}
+        >
+          {title}
+        </button>
+
+        <p>
+          {id === selectedTabId && content}
+        </p>
```

Comment:
Tab content should be rendered at the same place for all tabs, like in the example gif.
Hint - move it out of map `method`